src/route/mod.rs

```diff
@@ -777,4 +622,30 @@ pub struct Location {
     /// service_other. The default value is service_other so that preferred_side will not be
     /// disabled for any edges.
     pub street_side_cutoff: Option<f64>,
-}
\ No newline at end of file
+}
+
+#[cfg(test)]
+mod test {
+    use super::*;
+    #[test]
+    fn serialisation_snapshots() {
+        let manifest = Manifest::default();
+        insta::assert_json_snapshot!(manifest, @r#"
```

Comment:
Can you avoid the insta dependency here? We already depend on serde_json, so something like this should work:

assert_eq!(manifest, serde_json::json!({ ... }));

Reply:
If you want, I can do so..

The reason why I think that insta is a better solution though is what happens if it breaks

|  |  |
|--------|--------|
| `insta::assert_json_snapshot!(manifest` | ![image](https://github.com/user-attachments/assets/92a27a44-308b-4af6-bbdb-4cae414e1f3a) | 
| `assert_eq!(serde_json::to_value(manifest).unwrap(), serde_json::json!({..})` | ![image](https://github.com/user-attachments/assets/9a38af4f-7c31-4f98-9a65-53fdf54c6927) | 

Accepting the snapshots is also much simpler via `cargo insta review`

Reply:
Thanks, I would actually prefer to avoid insta - especially since this is a single test.